packages/cursorless-vscode-e2e/suite/fixtures/recorded/languages/perl/changeClassName.yml

```diff
@@ -0,0 +1,22 @@
+languageId: perl
+command:
+  version: 4
+  spokenForm: change class name
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: className}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: $var = Some::Package::func()
+  selections:
+    - anchor: {line: 0, character: 17}
+      active: {line: 0, character: 17}
+  marks: {}
+finalState:
+  documentContents: $var = ::func()
```

Comment:
"class name" should be the name of a class you're declaring, not one you're accessing. 

In general "name" is only for the name of something you're declaring, eg a variable, function, etc, not when you're accessing it

Reply:
This is fixed too.